### PR TITLE
Cleanup menus_ingame.cpp

### PR DESCRIPTION
**Pull request by Ryozuki**
Kinda progress for #1319 since TW compatibility isn't an issue now I think we could start doing it.

I removed useless comments too. (did this manually btw)

Imho we should have some clear coding style somewhere in the repo and not rely on the teeworlds hosted code style which i always forget to find, i would specially like a rule to avoid "1+1*3\*4" instead should be "1 + 1 * 3 * 4" lot more clean.

### CODE REVIEW

**Comment by def-** [comment]
I'm against removing curly braces on single-line ifs. The rest ist probably fine.

**Comment by Ryozuki** [comment]
Why? imho its cleaner

**Comment by def-** [comment]
https://www.imperialviolet.org/2014/02/22/applebug.html

**Comment by Ryozuki** [comment]
Not my fault they dont know non-brace if is just for 1 line :see_no_evil:

**Comment by def-** [comment]
This commonly happens in automatic merge conflict resolution, not just because of human error.

File: src/game/client/components/menus_ingame.cpp [hunk 1]
```diff
@@ -1111,23 +1051,19 @@ void CMenus::RenderGhost(CUIRect MainView)
...
-			{
-				CTextCursor Cursor;
-				TextRender()->SetCursor(&Cursor, Button.x, Button.y + (Button.h - 12.0f * UI()->Scale()) / 2.f, 12.0f * UI()->Scale(), TEXTFLAG_RENDER|TEXTFLAG_STOP_AT_END);
-				Cursor.m_LineWidth = Button.w;
-
-				TextRender()->TextEx(&Cursor, pItem->m_aPlayer, -1);
-			}
-			else if(Id == COL_TIME)
-			{
+			else if(Id == COL_NAME || Id == COL_TIME) {
```
Comment (heinrich5991) [inline src/game/client/components/menus_ingame.cpp]: Brace went up a line here.

**Comment by heinrich5991** [comment]
Uncontroversial: Spaces around operators. Removing dead code in comments.

I dislike removing braces around ifs.

`nullptr` isn't really used in the TW code base, I don't really think it provides much benefit there(?).

The foreach loops are also not really used in TW code, but I do think that they can improve readability, but might still be a good idea to check with the other devs.

The use of `auto` is also new here, not sure if we want this…

**Comment by def-** [comment]
I think general clean-up changes would first require a discussion about what makes sense, and best should be done automatically using clang-format and clang-tidy checks. It's not really worth it to manually go through code to clean up slight things like that.

**Comment by Ryozuki** [comment]
> 
> 
> I think general clean-up changes would first require a discussion about what makes sense, and best should be done automatically using clang-format and clang-tidy checks. It's not really worth it to manually go through code to clean up slight things like that.

I guess, but we should discuss that someday and not forget it :P 

I'll close this for now.

**Pull request closed without merging**